components/user/CourseCard.tsx

```diff
@@ -1,8 +1,7 @@
-import useCourse from "@hooks/useCourse"
-import Link from "next/link"
+import useCourse from '@hooks/useCourse'
+import Link from 'next/link'
 
-
-const CourseCard = ({ code }: { code: string }) => {
+const CourseCard = ({ code }: { code?: string }) => {
```

Comment:
Why did you make `code` optional?

Reply:
I was getting run time errors when calling CourseCard and something about something being undefined. So I changed this to optional. Turns out I just needed the course code to be lowercase. Everything works without making this optional; I just forgot to remove it. I will remove it now.